### DISCUSSION TITLE
It would be cool there is a mode for a image enhance like krea or magnific

### DISCUSSION THREAD

**Discussion by PierrunoYT** [upscayl/upscayl#909] [Ideas]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Description

It would be cool there is a mode for a image enhance like krea or magnific

**Comment by NayamAmarshe** [comment]
Duplicate of #623
